acceptable/lint.py

```diff
@@ -221,29 +221,26 @@ def check_custom_attrs(name, old, new, new_api=False):
     if 'description' not in new:
         yield LintWarning(name + '.description', 'missing description field')
 
-    if not new_api:
-        introduced_at = new.get('introduced_at')
-        if introduced_at is None:
-            if not old:  # new field
-                yield LintFixit(
-                    name + '.introduced_at', 'missing introduced_at field')
-        else:
-            introduced_at_changed = False
-            old_introduced_at = old.get('introduced_at')
-            if old_introduced_at is not None:
-                introduced_at_changed = old_introduced_at != introduced_at
-
-            if introduced_at_changed:
-                yield LintError(
-                    name + '.introduced_at',
-                    'introduced_at changed from {} to {}',
-                    old_introduced_at,
-                    introduced_at,
-                )
-            else:
-                # we have a specific introduced_at field, make sure to
-                # check it's referenced by a changelog entry
-                yield CheckChangelog(name, introduced_at)
+    # New field on existing api without introduced_at
+    if not new_api and not old and new.get('introduced_at') is None:
+        yield LintFixit(
+            '{}.introduced_at'.format(name), 'missing introduced_at field'
+        )
+    # Existing field but introduced_at has changed
+    elif (
+        not new_api and
+        old.get('introduced_at') is not None and
```

Comment:
This seems wrong, if you add an introduced_at to an existing field it should be a lint error, however the old code did this and it seem to be because old in this case is not False.

I think fixing this will require a bit more of a rewrite as I tried a simple change and many test broke :(